src/openrct2/game.h

```diff
@@ -17,6 +17,7 @@
 #ifndef _GAME_H_
 #define _GAME_H_
 
+#include "platform/platform.h"
```

Comment:
Because it includes system headers such as `windows.h` which adds lots of macros and causes problems with include order. It just causes lots of problems so its better not to. Its better to just hard code MAX_PATH. Eventually we will use `std::string` anyway.